kdwsdl2cpp/common/fileprovider.cpp

```diff
@@ -63,6 +65,17 @@ bool FileProvider::get( const QUrl &url, QString &target )
       return true;
   }
 
+  QString importPath = Settings::self()->importPath();
+  if (!importPath.isEmpty()) {
+      QDir importDir(importPath);
+      QString path = importDir.absoluteFilePath(url.host() + QDir::separator() + url.path());
+      if(QFile::exists(path)) {
+          qDebug("Using import path '%s'", qPrintable(path));
```

Comment:
I think this should be removed, or made more complete by printing url as well, to explain which file was found locally. If there are many files to look up, this output isn't very useful.

Reply:
Here I matched the other debug output. When the file will be downloaded, then it will be displayed as well. 
The URL is shown on the debug line before this. It will look something like this:
````
importing schema at http://docs.oasis-open.org/wsrf/bf-2.xsd
Using import path '/home/casper/QtProjects/onvifviewer/3rdparty/wsdl/docs.oasis-open.org//wsrf/bf-2.xsd'
````

Reply:
Yes I got confused by what's in "path", sorry.